app/build.gradle.kts

```diff
@@ -45,7 +45,8 @@ application {
 java {
     modularity.inferModulePath.set(true)
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(17))
+        vendor.set(JvmVendorSpec.ORACLE)
```

Comment:
Probably at now Oracle distributive can be not find by gradle, we can use AZUL, it's works fine on GitHub CI

Reply:
Removed the requirement for a specific vendor